doc-content/enterprise-only/project-data/reviewing-change-requests-proc.adoc

```diff
@@ -18,7 +18,8 @@ image::project-data/change-branch1.png[Selecting a branch menu]
 .. Review the *Overview* tab for general information about the change request.
 .. Click the *Changed Files* tab and expand a file to review the proposed changes.
 . Click a button in the top right corner.
```

Comment:
@hmanwani-rh , the sub steps within this step "Click a button ..." need to be bullets instead of arabic numeral substeps, because it's a list of button options/actions, not a sequence of steps.